### DISCUSSION TITLE
【开放性问题】最近有两篇关于 folding 的 paper，hypernova 和 protostar，想听听大家的看法

### DISCUSSION THREAD

**Discussion by flyq** [zkp-co-learning/zkp-co-learn#85] [Q&A（提问题在隔壁~）]
Hypernova: https://twitter.com/srinathtv/status/1650602409343991808?s=20
Protostar: https://twitter.com/benediktbuenz/status/1653025900915728385?s=20

期待大家从任何方面展开自己的想法，比如他们的性能提升相对之前的协议有多大，对一些做 zkEVM 的项目方来说是不是会影响到他们的技术路线之类的。

**Comment by gy001** [comment]
zkevm 2.0
